### DISCUSSION TITLE
recommendations for stuff to add to WebTV HD

### DISCUSSION THREAD

**Discussion by Selvenity** [SKCro/WebTV-HD#11] [Ideas]
**all the ones in bold are for the TV section.**
_The ones in.... THIS format is for the general stuff._
### the ones that are disguised as heading are just general recommendations. stuff after the header is NOT apart of a section.

**TV Listings! PLEASE I JUST WANNA SEE THE TV LISTINGS :(**
**TV Sites. could be useful.**
_WE WANNA SEE THE NOTICE OF EVICTION EMAIL DARN IT! I'VE WAITED TOO LONG >:(_
### allow for more users :)

**Comment by Selvenity** [comment]
oh and....

**add a page in WebTV HD that tells about the update! IM TIRED OF GOING TO THE GITHUB PAGE TO SEE WHAT THE NEW UPDATE ADDED**

**Comment by SKCro** [comment]
TV Listings is impossible to reimplement.
TV Sites is also impossible to reimplement. There would also be no point.
The eviction notice email will be coming next update.
More users is also impossible due to the lack of a backend.
The [GitHub commit log](https://github.com/SKCro/WebTV-HD/commits/main/) already acts as an update page. Why should I have to create one within WebTV HD itself?

**Reply by Selvenity** [reply]
Why would i go to the repo when i'm already on the page? Just takes too long.

**Reply by SKCro** [reply]
I'll add a link to the commit log on the home page.

**Comment by Selvenity** [comment]
You could use a XML file for listings (if you wanna know what i mean, see the inside of the "telivision-simuator-99" repo made by someone)

**Reply by SKCro** [reply]
I'd have to write an XML parser, which I don't want to do. Besides, I'd also have to pour a ton of time into perfecting the styles and layout and stuff. I don't really feel like doing all that just for TV listings that won't even be useful for anyone but you.

**Comment by Catrious** [comment]
You could add a setup of WebTV (where you choose the webtv email, choose username etc.)
